src/Extend/ApiController.php

```diff
@@ -216,6 +217,27 @@ public function setSort(array $sort, $callback = null)
         return $this;
     }
 
+    /**
+     * Eager loads relationships.
+     *
+     * First level relationships will be loaded regardless of them being included in the request.
```

Comment:
This is because they're needed for serializer logic, right? If so, can we mention that?